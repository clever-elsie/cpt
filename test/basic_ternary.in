let ternary1 = 1<2?10:20;
let ternary2 = 2>3?100:200;

\print(ternary1, ternary2); 