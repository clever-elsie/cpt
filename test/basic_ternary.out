10
200
2
